### PR TITLE
Multi video enhancements

### PR DESCRIPTION
**Pull request by chenrygit**

### CODE REVIEW

File: new-components/video.html [hunk 1]
```diff
@@ -66,6 +69,7 @@
 		text-transform:uppercase;
 		padding-bottom:10px;
 		text-decoration:none;
+		overflow:hidden;
 	}
 	.v2 a.video-title:hover {
```
Comment (juliemtaylor) [inline new-components/video.html]: Is the `a` selector needed on the classes for these links?

You may want to add the property `cursor: pointer;` to the hovers. For some reason it isn't changing.

File: new-components/video.html [hunk 2]
```diff
@@ -403,6 +414,77 @@
...
+	<!--// W50 VIDEO COMPONENT WITH MULTI-VIDEO CAROUSEL //-->
+	<div class="video video-w50">
+
+		<!--// iFrames must have a unique numerical id, prefixed with 'player"  Increment the id number for each instance of a video component used on a page. //-->
+		<iframe class="video-embed" id="player2" src="https://www.youtube.com/embed/Z6JRidQkWsk?rel=0" frameborder="0" allowfullscreen></iframe>
+
+		<div class="video-content">
+
+		    <!--// AEM property limit of 60 characters //-->
+				<a href="#" class="video-title">Different Videos</a>
```
Comment (juliemtaylor) [inline new-components/video.html]: Where does this link go?

**Comment by juliemtaylor** [comment]
For the W50 multi-video component, I don't see a comp on the story so I'm not sure what Leigh intended here. My feedback is the proportions on the thumbnails are off a bit. What do you think of this presentation?
screen shot 2017-06-14 at 3 09 54 pm

File: new-components/video.html (hunk 2 above)
Comment (chenrygit) [inline new-components/video.html]: This link is in the existing single video version of the component.   It points to the same location as the 'Learn More' link.  Consequently the 'Learn More link is not displayed when multiple videos are being displayed.   I debated on removing the link for multi-video, but haven't came to a conclusion.  Might be best if we tell AEM to disable that link if no link URL is entered into by the content administrator.

**Comment by chenrygit** [comment]
There wasn't a comp on this component - I just coded it and then we worked through tweaks together.   

The thumbnails use background-size:cover, so they'll always find a way to fit.   The reason they are different proportions is to fill the extra space at the bottom of the fixed-height component - w66 had a lot of blanks pace to fill.

**Comment by chenrygit** [comment]
Also, we're not displaying the description text when multi-video carousel is displayed.   We may add the description text later, but that'll require more effort on the AEM side to reduce carousel heights when description text is used.

**Pull request merged**